scripts/analysis.py

```diff
@@ -917,32 +948,29 @@ def capacity_calc(governments, timestep, entry, exit_step):
         list of governments (countries)
     timestep: np.linspace
         list of timestep from 0 to simulation time
-    entry: list
-        power_cap, agentid, parentid, entertime
+    entry_exit: list
+        power_cap, agentid, parentid, entertime, exittime
         of all entered reactors
-    exit_step: list
-        power_cap, agentid, parenitd, exittime
-        of all decommissioned reactors
 
     Returns
     -------
     power_dict: dictionary
         "dictionary with key=government, and
         value=timeseries list capacity"
     """
+
     power_dict = collections.OrderedDict()
     for gov in governments:
         capacity = []
         cap = 0
         for t in timestep:
-            for enter in entry:
-                if (enter['entertime'] == t and
-                        enter['parentid'] == gov['agentid']):
-                    cap += enter['max(value)'] * 0.001
-            for dec in exit_step:
-                if (dec['exittime'] == t and
-                        dec['parentid'] == gov['agentid']):
-                    cap -= dec['max(value)'] * 0.001
+            for agent in entry_exit:
+                if (agent['entertime'] == t and
+                        agent['parentid'] == gov['agentid']):
+                    cap += agent['max(value)'] * 0.001
+                if (agent['entertime + lifetime'] == t and
```

Comment:
is there really a key "entertime + lifetime" ? It seems more logical for this to be agent['entertime'] + agent['lifetime']

Reply:
in line 518, the `entry_exit` variable queries the sqlitefile so that there is a column called `entertime + lifetime`. But if this is a potential source of confusion, I can change it.
```
   entry_exit = cur.execute('SELECT max(value), timeseriespower.agentid, '
                             'parentid, entertime, entertime + lifetime'
                             ' FROM agententry '
                             'INNER JOIN timeseriespower '
                             'ON agententry.agentid = timeseriespower.agentid '
                             'GROUP BY timeseriespower.agentid').fetchall()

    return capacity_calc(governments, timestep, entry_exit)```

Reply:
It's certainly confusing, but as long as it's not a mistake, this is fine.